docs/workshop/model_development.md

```diff
@@ -1,29 +1,37 @@
 # Model Development
 
+In this section, we will learn how to train an AIOps model from scratch. The model that we plan to train in particular is a Time to Merge prediction model which offers time estimates on when a Pull Request on a Git repository will be merged. To train such a model we look at historic pull request data of the repository (can also be extended to organization) and engineer features from the Pull Requests which can impact the time to merge of a Pull Request.
+
+To train such a model, we will be using a Jupyterlab environment and make use of jupyter notebooks. The following steps will walk you through how to train a Github time to merge model.
 
 ## Set Environment Variables
 
-This is an important step to get right for successfully being able to run the notebooks.
+This is an important step to get right for successfully being able to run the jupyter notebooks.
+
+* Go to [Jupyterhub](https://jupyterhub-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/hub/spawn) and select the ocp ci analysis image and the “Workshop” container size.
+
+If you do not have access to the Jupyterlab instance being used for this workshop, move to chapter on [Onboarding](./onboarding.md).
 
-* Go to [Jupyterhub](https://jupyterhub-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/hub/spawn) and select the ocp ci analysis image and the “Workshop” container size
 * Open the terminal and clone your forked Github repository in Jupyterhub.
 
 `git clone https://github.com/{YOUR-USERNAME}/ocp-ci-analysis`
 
-* Set up env variables at the root of the repository.
+If you do not have a Github account or have not forked the repository, move to chapter on [Set up Git environment](./git_setup.md).
+
+* Set up environment variables at the root of the repository.
+
+Once you have cloned the repository, [cd](https://linuxize.com/post/linux-cd-command/) into the repository and create a `.env` file.
```

Comment:
@oindrillac . I missed to mention previously. But I think we need to add a instruction to configure their git repo. 

```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```

Reply:
good point will add this